components/tooltip/test/tooltip.vdiff.js

```diff
@@ -146,7 +146,7 @@ describe('tooltip', () => {
 				{ x: 50, y: 100 }
 			].forEach(({ x, y }) => {
 
```

Comment:
Should we maybe be doing something more on the `@brightspace-ui/testing` in the conversion from test suite/name -> filename? Like just remove/replace certain characters so that nobody can accidentally cause this?

Reply:
Ah looks like you're already on it!